cmd/soroban-cli/src/commands/fees/default.rs

```diff
@@ -0,0 +1,129 @@
+use clap::{command, ValueEnum};
+
+use crate::{
+    commands::global,
+    config::{locator, network},
+    print::Print,
+    rpc,
+};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Config(#[from] locator::Error),
+    #[error(transparent)]
+    Serde(#[from] serde_json::Error),
+    #[error(transparent)]
+    Network(#[from] network::Error),
+    #[error(transparent)]
+    Rpc(#[from] rpc::Error),
+    #[error("Fee amount must be at least 100 stroops, but got {0}")]
+    FeeTooSmall(u32),
+    #[error("Invalid fee stats metric {0}: {1}")]
+    InvalidFeeStatsResult(String, String),
+}
+
+// `clap` converts variants to kebab-case (e.g., `FeeMetric::Max` -> `max`).
+#[derive(Clone, Copy, Debug, ValueEnum)]
+pub enum FeeMetric {
+    Max,
+    Min,
+    Mode,
+    P10,
+    P20,
+    P30,
+    P40,
+    P50,
+    P60,
+    P70,
+    P80,
+    P90,
+    P95,
+    P99,
+}
+
+#[derive(Debug, clap::Parser, Clone)]
+#[command(group(
+    clap::ArgGroup::new("Fee Source")
+    .required(true)
+    .args(& ["amount", "fee_metric", "clear"]),
+))]
+pub struct Cmd {
+    /// Set the default inclusion fee amount, in stroops. 1 stroop = 0.0000001 xlm
+    #[arg(long)]
+    pub amount: Option<u32>,
+
+    /// Set the default inclusion fee based on a metric from the network's fee stats
+    #[arg(long, value_enum)]
+    pub fee_metric: Option<FeeMetric>,
+
+    /// Clear the default inclusion fee setting
+    #[arg(long)]
+    pub clear: bool,
+
+    #[command(flatten)]
+    pub network: network::Args,
+
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+}
+
+impl Cmd {
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let printer = Print::new(global_args.quiet);
+
+        if std::env::var("STELLAR_INCLUSION_FEE").is_ok()
+            && std::env::var("STELLAR_INCLUSION_FEE_SOURCE").is_err()
+        {
+            printer.warnln("Environment variable STELLAR_INCLUSION_FEE is set, which will override this default inclusion fee.");
```

Comment:
1. we need to add `INCLUSION_FEE` to cmd/soroban-cli/src/env_vars.rs
2. there's used to be `STELLAR_FEE` as well. Is that still being used or did it get replaced by the inclusion fee env var?
3. we don't show warnings anywhere when there are multiple sources for an option. Instead, should we remove this and thing of something else that can be applied globally?